res/controllers/Numark-Mixtrack-Pro-scripts.js

```diff
@@ -810,7 +790,12 @@ NumarkMixTrackPro.toggleDirectoryMode = function(
     // Toggle setting and light
     if (value) {
         NumarkMixTrackPro.directoryMode = !NumarkMixTrackPro.directoryMode;
-
+        // https://manual.mixxx.org/latest/it/chapters/appendix/mixxx_controls.html#control-[Library]-focused_widget
```

Comment:
Let's use the global link, it'll redirect to local versions if required
https://manual.mixxx.org/latest/chapters/appendix/mixxx_controls.html#control-[Library]-focused_widget